api/python/provisioner/commands/auto_deploy.py

```diff
@@ -65,6 +65,7 @@ def deployment_validations(deploy_check):
         try:
             check_res = check_cmd().run(deploy_check)
         except Exception as exc:
+            logger.error("Deployment Failed.\n")
```

Comment:
Since there is no exception to log, `None` is getting added to the log instead. 
Used `logger.error` wherever we're raising exception in next line.

```
2021-02-22 23:48:12,291 - ERROR - FAILED: Deployment.
None
2021-02-22 23:48:12,291 - ERROR - PROVISIONER FAILED. Exiting..
```